qemu/tests/cfg/virtio_fs_hotplug.cfg

```diff
@@ -0,0 +1,88 @@
+- virtio_fs_hotplug:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1
+    type = virtio_fs_hotplug
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    kill_vm = yes
+    start_vm = yes
+    extra_filesystems = fs
+    fs_target = 'myfs'
+    fs_source_dir = virtio_fs_test/
+    fs_dest = '/mnt/${fs_target}'
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    test_file = 'test_file'
+    # create new sources if there is one already
+    force_create_fs_source = no
+    remove_fs_source = yes
+    fs_driver_props = {"queue-size": 1024}
+    mem = 4096
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = 4G
+    use_mem_mem1 = no
+    share_mem = yes
+    cmd_dd = 'dd if=/dev/urandom of=%s bs=1M count=2048 oflag=direct'
+    cmd_md5 = 'md5sum %s'
+    io_timeout = 120
+    s390, s390x:
+        required_qemu = [5.2.0,)
+        vm_mem_share = yes
+        pre_command_noncritical = yes
+        pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+        setup_hugepages = yes
+        kvm_module_parameters = 'hpage=1'
+        expected_hugepage_size = 1024
+        cmd_dd = 'dd if=/dev/urandom of=%s bs=1M count=200 oflag=direct'
+    !s390, s390x:
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    Windows:
+        # install winfsp tool
+        i386, i686:
+            install_path = 'C:\Program Files'
+        x86_64:
+            install_path = 'C:\Program Files (x86)'
+        driver_name = viofs
+        install_cmd = 'msiexec /i WIN_UTILS:\winfsp-1.7.20172.msi /qn'
+        check_installed_cmd = 'dir "%s" |findstr /I winfsp'
+        viofs_log_file = C:\\viofs_log.txt
+        viofs_sc_create_cmd = 'sc create VirtioFsSvc binpath="%s -d -1 -D ${viofs_log_file}" start=auto'
+        viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
+        viofs_sc_start_cmd = 'sc start VirtioFsSvc'
+        viofs_sc_query_cmd = 'sc query VirtioFsSvc'
+        virtio_win_media_type = iso
+        cdroms += " virtio"
+        cmd_dd = 'dd if=/dev/random of=%s bs=1M count=200'
+        cmd_md5 = "%s: && md5sum.exe %s"
+    variants:
+        - @default:
+        - with_repetition:
+            only plug.one_fs_source
+            repeat_times = 100
```

Comment:
Since there will be at least 2 iterations in the cases, could we repeat to reduce it a bit? 
Like 50 times?